assets/sys/src/reloivd/Makefile

```
@@ -0,0 +1,21 @@
+CXX = i386-vios-elf-g++
+CC  = i386-vios-elf-gcc
+
+SRC = src/main.cpp
+OBJ = build/main.o
+OUT = ../../../sbin/reloivd
+
+CFLAGS = -O2
+LDFLAGS  = 
+
+all: $(OUT)
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)